static/app/views/performance/newTraceDetails/traceTypeWarnings/useTransactionUsageStats.tsx

```diff
@@ -0,0 +1,67 @@
+import type {Organization} from 'sentry/types/organization';
+import {useApiQuery} from 'sentry/utils/queryClient';
+
+import type {TraceTree} from '../traceModels/traceTree';
+
+// 1 hour in milliseconds
+const ONE_HOUR = 3600 * 1000;
```

Comment:
```suggestion
const ONE_HOUR = 60 * 60 * 1000;
```
nit: this feels slightly more readable to me 😄